elements/lisk-transactions/src/utils/constructor.ts

```diff
@@ -0,0 +1,52 @@
+import { getAddressFromPublicKey } from '@liskhq/lisk-cryptography';
+
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+export const decideOnRecipientId = (
```

Comment:
`decideOnRecipientId` doesn't describe what it's doing.
Maybe `safeGetRecipientId`? (which doesn't sounds good too)